app/routes/closure.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs')
+const { closureAdmin } = require('../auth/permissions')
+const schema = require('./schemas/closure')
+const bulkSchema = require('./schemas/bulk-closure')
+const parsedSchema = require('./schemas/parsed-closure')
+const { post } = require('../api')
+
+module.exports = [{
+  method: 'GET',
+  path: '/closure',
+  options: {
+    auth: { scope: [closureAdmin] },
+    handler: async (_request, h) => {
+      return h.view('closure')
+    }
+  }
+},
+{
+  method: 'GET',
+  path: '/bulk-closure',
+  options: {
+    auth: { scope: [closureAdmin] },
+    handler: async (_request, h) => {
+      return h.view('bulk-closure')
+    }
+  }
+},
+{
+  method: 'POST',
+  path: '/closure',
+  options: {
+    auth: { scope: [closureAdmin] },
+    validate: {
+      payload: schema,
+      failAction: async (request, h, error) => {
+        return h.view('closure', { errors: error, frn: request.payload.frn, agreement: request.payload.agreement, day: request.payload.day, month: request.payload.month, year: request.payload.year }).code(400).takeover()
+      }
+    },
+    handler: async (request, h) => {
+      let day = request.payload.day.toString()
+      if (day.length !== 2) {
+        day = `0${request.payload.day}`
+      }
+      let month = request.payload.month.toString()
+      if (month.length !== 2) {
+        month = `0${request.payload.month}`
+      }
+      request.payload.date = `${request.payload.year}-${month}-${day}T00:00:00`
+      delete request.payload.day
+      delete request.payload.month
+      delete request.payload.year
```

Comment:
You don't really need to delete these as the `request` object isn't used again after this?